acquire/acquire.py

```diff
@@ -858,6 +858,20 @@ def get_spec_additions(cls, target: Target, cli_args: argparse.Namespace) -> Ite
         spec.update(("path", log_path) for log_path in chain(*iis_plugin.log_dirs.values()))
         return spec
 
+@register_module("--spse")
+class SPSE(Module):
+    DESC = "Windows Share Point Server (SPSE) logs"
+
+    @classmethod
+    def get_spec_additions(cls, target: Target, cli_args: argparse.Namespace) -> Iterator[tuple]:
+        spec = set()
+        key = "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Shared Tools\\Web Server Extensions\\16.0\\WSS"
```

Comment:
Can you glob or iterate over the version subkey? Can use the Exchange or MSSQL module as example.

Also linter is probably going to complain about only one newline above this.

Reply:
Yes. I assume you've tested this? Don't feel like spinning up a whole SharePoint system 😄.